scripts/barclamp_install.rb

```diff
@@ -54,10 +55,12 @@ def usage()
 
 usage if ARGV.length < 1
 
-tmpdir = "/tmp/bc_install-#{Process.pid}-#{Kernel.rand(65535)}"
+tmpdir = Dir.mktmpdir "component-install-"
```

Comment:
Yes, please remove all the `tmpdir` parts... They are totally useless :)
Especially the remove instructions for tmpdir as it's `Dir.mktmpdir` anyway now.